RELEASING.md

```diff
@@ -87,31 +89,33 @@ To push a release version
   git commit -m "Release [VERSION HERE]"
   ```
 
-6. Make sure you are using Maven version 3.3 or higher to support the Nexus plugin required to stage a release.
+6. Create and merge in a PR to reflect the updated project version.
+
+7. Put the repo in a clean state: a) `mvn clean`, then b) `git clean -x -f -d`.
```

Comment:
I don't think we need `mvn clean`. `git clean` should clean everything.

Reply:
Done